NEWS.md

```diff
@@ -2,6 +2,9 @@
 
 # ggplot2 3.3.0
 
+* Added an `outside` option to `annotation_logticks()` that places tick marks
+  outside of the plot bounds. (#3783, @kbodwin)
```

Comment:
Thanks for making this change. Unfortunately I'll have to request one more modification. I think your news item is in the wrong location in the News file. This feature won't make it into ggplot2 3.3.0, as that version has already been branched off.

Could you move your entry to *above* the ggplot2 3.3.0 heading?

Reply:
oops, sorry!